pkg/gitops/flux/installer.go

```diff
@@ -219,6 +243,27 @@ func (fi *Installer) Run(ctx context.Context) error {
 	return nil
 }
 
+func continueInstallation() (bool, error) {
+	for {
+		reader := bufio.NewReader(os.Stdin)
+		fmt.Print("Continue? [y/n]: ")
```

Comment:
Uhm, I am not sure we should be ask the user about this. I would lean towards:

1. Deleting every resource we were to create first (as opposed to deleting the whole namespace since it can contain more stuff)
2. Do it without asking for permission (we shouldn't ask for permission when using `kubectl apply` which would have an overall similar impact).

Reply:
Also, I wouldn't overwrite the ssh-key secret (`kubectl apply` would leave it as is, since it's empty in the manifest and it's convenient not needing to overwrite the ssh key every time you reinstall).

Reply:
Grrr, 1. is what I had initially 🤦‍♂

Reply:
Good point about the SSH key 👍

Reply:
As I mentioned in slack, we should always assume non-interactive use-cases. None of other commands prompt user for anything. This is one of the things I'd like to add to CLI design guidelines. We should make softer behaviour  the default and provide a flag to override (but only if there is strong use-case for such flag).